deployments/data-hub/data-hub-configs.yaml

```diff
@@ -47,22 +47,43 @@ data:
   gmail-data-pipeline.config.yaml: |-
     projectName: 'elife-data-pipeline'
     datasetName: '{ENV}'
-    gmailLabelData:
```

Comment:
Just want to check this config file change isn't going to break existing prod image?

Reply:
Also well spotted.

Reply:
Gmail pipeline in production is not working today after seeing the staging works after I will make same changes on production too. Is there any other suggestion I can do, maybe changing prod or staging both together?

Reply:
Added new version of gmail config file